app/src/main/res/values/styles.xml

```diff
@@ -8,6 +8,14 @@
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <!-- Base application theme. -->
```

Comment:
Could we use the base `AppTheme` instead of creating this new one 🤔

Reply:
In the `NewsDetailsActivity` the `ActionBar` is set using **compose**, so I wanted to hide the default `ActionBar` which the activity provides from the theme. Can we can use the same theme and hide the `ActionBar` from the theme in `onCreate` using `supportActionBar?.hide()`?

Reply:
Ah I see. Got it now, in that case it's best to keep to have new theme like you did and perhaps update the code comment above it saying "Base theme for Compose"